README.md

```diff
@@ -97,6 +97,14 @@ Check [capabilities](https://www.browserstack.com/automate/capabilities) in Brow
 
 #### Other configurations
 
+##### Screen resolution
+
+Specify the screen resolution for the machine in browserstack using the options `screen_height` and `screen_width`
+
+```ruby
+Browsenator.for(:chrome, remote: :browserstack, screen_width: 1920, screen_height: 1200)
+```
+
```

Comment:
Here maybe a note to let the users know not all resolutions are valid and that they need to check in browserstack. We can include a link to capabilities generator, there is one in the readme.

Reply:
I will add that in :D